spec/jobs/cancel_reservations_job_spec.rb

```diff
@@ -1,19 +1,14 @@
 require 'rails_helper'
 
 RSpec.describe CancelReservationsJob, type: :job do
-  subject(:run_job) { described_class.perform_later(screening_id) }
-  let(:screening_id) { 1 }
-
-
-  
-
-  before do
-    allow( Reservations::UseCases::CancelUnpaidReservations).to receive(:call)
-    run_job
-  end
+  let(:cinema_hall) {create(:cinema_hall)}
+  let(:movie) {create(:movie)}
+  let(:screening) {create(:screening, date: Time.current - 30.minutes, movie: movie, cinema_hall: cinema_hall) }
+  let(:cancel_unpaid_reservations) { instance_double(Reservations::UseCases::CancelUnpaidReservations, call: true) }
 
```

Comment:
```
before do
allow(Reservations::UseCases::CancelUnpaidReservations).to receive(:new).and_return(cancel_unpaid_reservations)
end
```

Reply:
I change it and it gives me the same error :(